pkg/virt-handler/vm.go

```diff
@@ -874,19 +874,22 @@ func (d *VirtualMachineController) updateVMIStatus(origVMI *v1.VirtualMachineIns
 			// Set Pod Interface
 			interfaces := make([]v1.VirtualMachineInstanceNetworkInterface, 0)
 			for _, network := range vmi.Spec.Networks {
-				if network.NetworkSource.Pod != nil {
-					podIface, err := d.getPodInterfacefromFileCache(vmi, network.Name)
-					if err != nil {
-						return err
-					}
+				podIface, err := d.getPodInterfacefromFileCache(vmi, network.Name)
+				if err != nil {
+					return err
+				}
+
+				if podIface != nil {
```

Comment:
I also had to add this check, because sometimes, [at least fior SRIOV](https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-handler/vm_test.go#L2709) , we don't always cache the iface

Reply:
I am not sure, but that's why this `if` is required